src/main/webapp/i18n/en/complaintResponse.json

```diff
@@ -1,5 +1,15 @@
 {
     "artemisApp": {
+        "locks": {
+            "acquired": "You now have the lock on this complaint for 24 hours!",
+            "removeButton": "Remove lock",
+            "lockRemoved": "Removed lock from complaint",
+            "lockInformation": "Locked by {{user}} until {{endDate}}",
+            "lockInformationYou": "Locked by YOU until {{endDate}}",
+            "notUnlocked": "Not Locked",
+            "lockStatus": "Lock Status",
+            "notAllowedToRespond": "You are not allowed to respond to the complaint or more feedback request!\nFor team exercises, the team tutor is the assessor and handles both complaints and feedback requests himself.\nFor individual exercises, complaints are handled by a secondary reviewer and feedback requests by the assessor himself\nFor exam test runs, the original assessor is allowed to respond to complaints"
```

Comment:
This seems to be missing a fullstop after the second `himself`. (And at the end?)

Reply:
Fixed in 679c7b042270be27e77df762d438a0573023ece2